chidioguejiofor/movies-app
# Prof: Created comments page

#### What does this PR achieve?
- adds a CommentLayout for the comments page
- extends the routing to handle pages that are not found
- connects the comments infrastructure to firebase
- ran linting

#### Any background context?
- This is built in such a way that the comments are retrieved in real-time

#### Manual Testing Steps
- Go to a comments page
- Add a comment 
- You should see the new comment
- Copy the URL in a new tab and add another comment 
- You should see the new comment in the previous tab

<img width="1680" alt="Screenshot 2020-11-23 at 3 30 24 PM" src="https://user-images.githubusercontent.com/38565349/99974141-d4044900-2da0-11eb-8e7f-35175cb66da1.png">
